### ISSUE TITLE
Add the Basic Static Website in Web Development Folder

### ISSUE THREAD

**Issue by rohansaini886** [rohansaini886/Hacktoberfest2022#1]
You Can start with basic HTML5 webpages.

**Comment by Hussain6** [comment]
Hi Sir can I also contribute to this repository?

**Comment by flow6979** [comment]
Hey, I am very much interested in contributing to this repository. can I add any basic website like portfolio, etc? or if required you can tell me the type of work you want me to contribute.
 Thank you

**Comment by saibharath295** [comment]
hi @rohansaini886  can I contribute to your issue, if yes let me know because I am learning web dev
thank you

**Comment by rohansaini886** [comment]
yes @saibharath295 and @flow6979 you can add your codes here. But make sure that you have commit it after the 1st October because your "Hacktoberfest-Accepted" tag count onwards it.

**Comment by rohansaini886** [comment]
@aditya-464 There is no need to raise a new issue just fork this repo. And add your website in the folder and just make PR for it.

**Comment by aditya-464** [comment]
@rohansaini886  Got it!

**Comment by Rakteem007** [comment]
Can I contribute to this issue @rohansaini886

**Comment by Prajwal0225** [comment]
@rohansaini886 Can I include the Steve Jobs timeline static webpage that I created when I was learning HTML?

You can view the page at
`https://prajwal0225.github.io/Steve-Jobs-TributePage/`

**Comment by Souvik-Nayak** [comment]
@rohansaini886 I want to work on this issue and i am thinking of implementing timer static page or a color picker webapp line "colorhunt" which will help to to get the HEX code of a certain color easily !

**Comment by rahulaarora** [comment]
Can I add my portfolio website template?

**Comment by rahulaarora** [comment]
> Can I add my portfolio website template?

@rohansaini886

**Comment by sumitbaroniya** [comment]
Basic website added
